examples/disaggregated/slurm/run_benchmark.sh

```
@@ -89,31 +89,31 @@ do_get_logs(){
+        --dataset-name random \
+        --dataset-path ${shared_gpt_path} \
+        --random-input-len ${isl} \
+        --random-output-len ${osl} \
+        --random-prefix-len 0 \
+        --num-prompts ${max_count} \
+        --max-concurrency ${concurrency} \
+        --host ${hostname} \
+        --port ${port} \
+        --ignore-eos \
+        --no-test-input \
+        $(if [ "${streaming}" = "false" ]; then echo "--non-streaming"; fi)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)